enterprise/frontend/src/metabase-enterprise/moderation/index.js

```diff
@@ -32,33 +33,60 @@ Object.assign(PLUGIN_MODERATION_SERVICE, {
   isRequestDismissal,
   getModerationEvents,
   isRequestOpen,
+  getReviewType,
 });
 
-export function getModerationIssueActionTypes(isModerator, moderationRequest) {
-  return isModerator
-    ? getModerationReviewActionTypes(moderationRequest)
-    : getModerationRequestActionTypes();
+export function getModerationIssueActionTypes(userType, targetIssueType) {
+  switch (userType) {
+    case USER_TYPES.user:
+      return [
+        [
+          ACTIONS.verification_request.type,
+          ACTIONS.something_wrong.type,
+          ACTIONS.confused.type,
+        ],
+      ];
+    case USER_TYPES.moderator:
+      return [
+        [
+          ACTIONS.verified.type,
+          ACTIONS.misleading.type,
+          ACTIONS.confusing.type,
+        ],
+        !isRequestType(targetIssueType) && [
+          ACTIONS.verification_request.type,
+          ACTIONS.something_wrong.type,
+          ACTIONS.confused.type,
+        ],
+        getDismissalAction(targetIssueType),
+      ].filter(Boolean);
```

Comment:
nip: got a little stuck at `!isRequestType(...) &&`. When I look at the code, I think "okay, these actions will be displayed if I'm a moderator and the issue is not a moderation request, but what instead?". Maybe replacing it with some "positive" check would improve readability. Is it for an `isReviewType` case maybe?

Reply:
agree, it is confusing. I'm gonna punt on changing it because, after a recent product meeting, I think this function will end up going away entirely. If not, it will get simplified a lot in the process of redesigning.